.github/workflows/cppa-clang-ci.yml

```
@@ -0,0 +1,81 @@
+name: cppa-clang CI
+
+on:
+  pull_request:
+    paths:
+      - 'clang/**'
+      - 'llvm/**'
+      - 'cmake/**'
+      - '.ci/**'
+      - '.github/workflows/cppa-clang-ci.yml'
+    paths-ignore:
```

actionlint (1.7.11)

[error] both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)